build/image/photon/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.10 as golang-build
+FROM golang:1.21.5 as golang-build
```

Comment:
After upgrade golang to 1.21, you should be care is there is glibc issue.
https://github.com/vmware-tanzu/nsx-operator/pull/448
if yes, you should disable CGO.

Reply:
ok